test/e2e/nodepool_test.go

```diff
@@ -125,6 +125,10 @@ func TestNodePool(t *testing.T) {
 						name: "OpenStackAZTest",
 						test: NewOpenStackAZTest(ctx, mgtClient, hostedCluster),
 					},
+					{
+						name: "OpenStackImageTest",
```

Comment:
The e2e job timeouted, that makes me think we have too many tests maybe. We could combine this scenario with another one (AZ or multi net for example).